backend/api/v1/endpoints/Research/PostResearch.py

```
@@ -0,0 +1,39 @@
+            title=title,
+            research_field=research_field,
+            details=details,
+            creator_id=current_user.id
+        )
+        db.add(new_research)
+        db.commit()
+        db.refresh(new_research)
+
+        return {"message": "Research work posted successfully", "research_id": new_research.id}
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)